### ISSUE TITLE
version 0.31.4 es build exports are broken

### ISSUE THREAD

**Issue by jharris4** [react-bootstrap/react-bootstrap#2860]
I just tried out `0.31.4` and I get the error:

```
ReferenceError: Can't find variable: exports
(anonymous function) — CloseButton.js:57
```

The last 2 lines of `react-bootstrap/es/CloseButton.js` look like this:

```
export default CloseButton;
module.exports = exports['default'];
```

Same seems to be true of all the `.js` files in the `es` directory.

Perhaps this is related to the `add-module-exports` babel plugin???

**Comment by karlshea** [comment]
Looks like this is being worked on in #2856

**Comment by jharris4** [comment]
@karlshea was just going to comment with the same thing. Does indeed look like it's in progress

**Comment by richtera** [comment]
Same problem here.
![image](https://user-images.githubusercontent.com/708186/31965641-f617eda0-b8d5-11e7-9c48-5414a39df20b.png)

**Comment by richtera** [comment]
I can't get add-module-exports to work, no.

**Comment by richtera** [comment]
NOTE: Maybe unpublish the npm since ^0.31.3 will pull it in the next time someone does npm install?

**Comment by taion** [comment]
Merged and published as 0.31.5

**Comment by richtera** [comment]
Works! Thank you.
